packages/cypher-builder/src/Cypher.ts

```diff
@@ -107,7 +114,9 @@ export type { CompositeClause } from "./clauses/utils/concat";
 
 // utils
 import { escapeLabel } from "./utils/escape-label";
+import { compileCypherIfExists } from "./utils/compile-cypher-if-exists";
 
 export const utils = {
     escapeLabel,
+    compileCypherIfExists
```

Comment:
I thought the same, on the other hand I thought that it could be helpful when used with `RawCypher`, for instance see this [case](https://github.com/neo4j/graphql/pull/2115/files#diff-5b6a5960afaf308bd4db532ba733a1f24dc8d8ed3c6426230b5dc9f378f5a219R121-R126).
What do you think?